contracts/libraries/Oracle.sol

```diff
@@ -270,9 +276,12 @@ library Oracle {
                 beforeOrAt.tickCumulative +
                     ((atOrAfter.tickCumulative - beforeOrAt.tickCumulative) / observationTimeDelta) *
                     targetDelta,
-                beforeOrAt.liquidityCumulative +
-                    ((atOrAfter.liquidityCumulative - beforeOrAt.liquidityCumulative) / observationTimeDelta) *
-                    targetDelta
+                beforeOrAt.secondsPerLiquidityCumulativeX128 +
```

Comment:
this one hurt my brain

Reply:
yeah technically we could do tick cumulative the same way (multiply and then divide) but the result woudl be th same